frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -90,7 +90,7 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
                 )
               }
               {
-                !hasRole(currentUser, "ROLE_DRIVER") && isParticipant(currentUser) && (
+                hasRole(currentUser, "ROLE_ADMIN") && isParticipant(currentUser) && (
```

Comment:
Do we need the && isParticipant here?

I think that if you are an admin, you are automatically a participant, no?

Reply:
Yes, it appears the && isParticipant check is redundant. I updated the code.